frontend/packages/dapple/package-post-init.js

```diff
@@ -47,13 +45,14 @@ const tokenSpecs = {
   VSL: { precision: 18, format: '0,0.00[0000000000000000]' },
   PLU: { precision: 18, format: '0,0.00[0000000000000000]' },
   MLN: { precision: 18, format: '0,0.00[0000000000000000]' },
+  RHOC: { precision: 9, format: '0,0.00[0000000]'  }
```

Comment:
Well indeed it's 8 but if you have supply of 1 000 000 000  so the minimum you can have is 1 / 1 000 000 000 which is 0.000000001 right?